.github/workflows/manubot.yaml

```diff
@@ -0,0 +1,40 @@
+name: Manubot
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    name: Build Manuscript
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Install Environment
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: manubot
+          environment-file: build/environment.yml
+          auto-activate-base: false
+          miniconda-version: 'latest'
+      - name: Build Manuscript
+        env:
+          GITHUB_PULL_REQUEST_SHA: ${{ github.event.pull_request.head.sha }}
+        shell: bash --login {0}
+        run: bash build/build.sh
+      - name: Upload Artifacts
```

Comment:
Will this eventually replace the AppVeyor artifacts and notification messages?

Reply:
The main difference now is that it doesn't provide notification messages (possibly it could not sure). Although I'm not sure I love the messages since they get overwhelming.

So yeah, I think it is a satisfactory substitute, but requires users are aware of the feature, since you have to navigate to it.